docs/argocd.md

```diff
@@ -149,6 +149,27 @@ Just like that, the developer who controls the helm chart is able to request any
 of a secret in the vault. Therefore, the cluster administrator must restrict the access to the secrets for specific applications. This is achieved by following this procedure :
 
 1. Create a specific policy in Vault for the application which only gives access to the secrets needed by the application
+
+RW policy example :
+
+```hcl
+path "kv/metadata/my_app/*" {
+  capabilities = ["list", "read", "delete"]
+}
+path "kv/data/my_app/*" {
+  capabilities = ["create", "update", "read", "delete"]
+}
+path "kv/delete/my_app/*" {
+  capabilities = ["update"]
+}
+path "kv/undelete/my_app/*" {
+  capabilities = ["update"]
+}
+path "kv/destroy/my_app/*" {
+  capabilities = ["update"]
+}
```

Comment:
When I read the documentation, even it's an example, the paths ``kv/delete``, ``kv/undelete`` and ``kv/destroy`` could be confusing for someone who start with Vault. 

I think it's better to keep the 2 first path in the policy, it's more realistic. and delete the other.

Reply:
From my understanding, this 3 actions can be performed from the vault UI for example to delete a version, without deleting the secret.

`delete` is used when you want to be able to rollback with undelete
`destroy` is used to permanently remove a version of a secret, or the secret itself

I think `destroy` is required because we want to allow maintainers to clean their secrets architecture. The `delete`/`undelete` can be challenged, because we could "force" the user to `destroy` instead of `delete` it, but I am not comfortable to disable native features to users like this.

Reply:
You're right. I just read this (https://github.com/jeffsanicola/vault-policy-guide). 

I doesn't understand the specific path ``kv/destroy`` ``kv/undelete`` and ``kv/delete`` was use for the destroy, restore and delete operation in the KV secret engine. I learn something, Thanks.